app/src/main/java/com/travelguide/fragments/TripPlanListFragment.java

```diff
@@ -168,6 +191,15 @@ public void done(List<TripPlan> tripPlans, ParseException e) {
         });
     }
 
+    private void setWhereClause(ParseQuery query) {
+        if (city != null && !city.equalsIgnoreCase("Any") && !city.equalsIgnoreCase(""))
```

Comment:
This could be simplified by this method for all three fields: http://developer.android.com/reference/android/text/TextUtils.html#isEmpty(java.lang.CharSequence)